src/lib/gql/client.ts

```diff
@@ -64,3 +67,20 @@ export const client = {
   mutate: thunder('mutation'),
   subscribe: thunder('subscription'),
 };
+
+export function useTypedSubscription<
+  Z extends ValueTypes[O],
+  O extends 'subscription_root'
+>(
+  subscription: Z | ValueTypes[O],
+  options?: SubscriptionHookOptions<InputType<GraphQLTypes[O], Z>>,
+  operationOptions?: OperationOptions
+) {
+  // @ts-ignore
```

Comment:
@CryptoGraffe these give long `stack depth` errors when compiling with the ts-ignores